Platform/Apple/virtual/src/font/fontEngine.s

```diff
@@ -394,50 +392,43 @@ DoAgn	PHA
 	STA (zTmp1),Y	;write to HGR. Use indrct, indxd adrssing
 	LDA Flg2nd	;check if pixel pattern crosses 2-bytes
 	BEQ Chk8xcp	;if not, then skip to next line of bitmap
-	LDA #0
-	STA Flg2nd	;else, first, clear the flag
+	STY Flg2nd	;else, first, clear the flag
```

Comment:
>> @@ -394,50 +392,43 @@ DoAgn	PHA
>>  	STA (zTmp1),Y	;write to HGR. Use indrct, indxd adrssing
>>  	LDA Flg2nd	;check if pixel pattern crosses 2-bytes
>>  	BEQ Chk8xcp	;if not, then skip to next line of bitmap
>> -	LDA #0
>> -	STA Flg2nd	;else, first, clear the flag
>> +	STY Flg2nd	;else, first, clear the flag
>
> Are you sure about this change? On the second and subsequent iterations of
> the `DoAgn` loop, Y will be non-zero and this would not clear `Flg2nd`

The BEQ will be taken on the second pass.  There are no further loops
to consider.